backend/src/main/game/monster/concrete_monster_cards/wild.py

```diff
@@ -0,0 +1,20 @@
+from backend.src.main.game.monster.random_monster_card import RandomMonsterCard
+from backend.src.main.game.monster.values import DungeonCardValues, NumberedRoomTileValues
+
+
+class Wild(RandomMonsterCard):  # pylint: disable=too-few-public-methods
+
+    def __init__(self):
+        map_values = {NumberedRoomTileValues.ONE: DungeonCardValues.MONSTER,
```

Comment:
curly brace usage should be consistent, either same line or next line

Reply:
This has been resolved